osu.Game/Overlays/BeatmapSet/Header.cs

```diff
@@ -68,8 +66,24 @@ private void updateDisplay()
                 downloadButtonsContainer.FadeIn(transition_duration);
                 favouriteButton.FadeIn(transition_duration);
 
-                noVideoButtons.FadeTo(BeatmapSet.OnlineInfo.HasVideo ? 0 : 1, transition_duration);
-                videoButtons.FadeTo(BeatmapSet.OnlineInfo.HasVideo ? 1 : 0, transition_duration);
+                if (BeatmapSet.OnlineInfo.HasVideo)
+                {
+                    videoButtons.Children = new[]
+                    {
+                        new DownloadButton(BeatmapSet),
+                        new DownloadButton(BeatmapSet, true),
+                    };
+
+                    videoButtons.FadeIn(transition_duration);
+                    noVideoButtons.FadeOut(transition_duration);
```

Comment:
This is just a question out of curiosity: Wouldn't it work with just a `FillFlowContainer`? Or does the `Spacing` that is set later make a visual difference for 1 vs 2 buttons?

Reply:
Not entirely sure what you mean here, `videoButtons` and `noVideoButtons` are already `FillFlowContainer`s.

Reply:
No. `noVideo` is just a `Container`. I just thought you could simplify this a bit if you use only one `FillFlowContainer` instead of two. But I don't know if that would work because I'm not sure about the reason to have it split.